search/includes/classes/class-health.php

```diff
@@ -239,6 +239,11 @@ public static function validate_index_posts_content( $start_post_id = 1, $last_p
 				$last_post_id = self::get_last_post_id();
 			}
 
+			// Cleanup WordPress object cache to keep memory usage under control
+			global $wp_object_cache;
+			$wp_object_cache->flush();
+			$wp_object_cache->group_ops = array();
```

Comment:
The array `group_ops` is not emptied by `flush()`, but it also increases in size during the command execution. That's why I'm emptying it manually.

Reply:
There's also a function provided by `\WPCOM_VIP_CLI_Command` that does this (and a few other things) called `\WPCOM_VIP_CLI_Command::stop_the_insanity()` - we should use that one to handle these.

Reply:
Oh! Sure, that's a better option, thanks! 😊